packages/apputils/package.json

```diff
@@ -42,12 +42,12 @@
     "@phosphor/signaling": "^1.2.2",
     "@phosphor/virtualdom": "^1.1.2",
     "@phosphor/widgets": "^1.5.0",
+    "@types/react": "^16.0.19",
```

Comment:
Sorry, a number of other packages depend on `~16.0.19`, so we shouldn't update to `^` without at least changing all the other packages. I'm okay with going back to `~`.